### ISSUE TITLE
Front end in MCP uses MCP Backend API (depends on #61)

### ISSUE THREAD

**Issue by abarciauskas-bgse** [NASA-IMPACT/delta-ui#133]
Depends on https://github.com/NASA-IMPACT/delta-backend/issues/61

**Comment by aboydnw** [comment]
Closing in favor of https://github.com/NASA-IMPACT/delta-backend/issues/96 as it feels more comprehensive.
